File: 0001 Code Chef/0018 Saving Taxes/implementation/0003 c#/SavingTaxes.cs
using System;


class SavingTaxes{
    static void Main(string[] args){
        string stringInput;
        stringInput = Console.ReadLine();
        int test = Convert.ToInt32(stringInput);
        while(test > 0){
            stringInput = Console.ReadLine();
            string[] stringArray = stringInput.Split(" ");
            int x = Convert.ToInt32(stringArray[0]);
            int y = Convert.ToInt32(stringArray[1]);
            if(x > y){
                Console.WriteLine(x-y);
            }else{
                Console.WriteLine(0);
            }
            test = test - 1;
        }
    }
}

File: 0001 Code Chef/0018 Saving Taxes/implementation/0008 python3/saving_taxes.py

if __name__ == "__main__":
    test = int(input())
    while(test > 0):
        string_input = input()
        string_array = string_input.split(" ");
        x = int(string_array[0]);
        y = int(string_array[1]);
        if(x > y):
            print(x-y)
        else:
            print(0)
        test = test - 1